src/components/Header/Header.js

```diff
@@ -15,6 +15,7 @@ const AppHeader = styled.header`
   left: 0;
   right: 0;
   padding: 8px;
+  z-index: 30;
```

Comment:
Please consider using css variables to keep track of all used `z-indices`. Could be in `GlobalStyles`.

Reply:
Applied variables for both z-index and color! :)